packages/mongo-nav/src/MongoNav.spec.tsx

```diff
@@ -92,63 +61,64 @@ describe('packages/mongo-nav', () => {
     });
 
     test('the organization and project navs are rendered', () => {
-      expect(expectedElements.orgNav).toBeInTheDocument();
-      expect(expectedElements.projectNav).toBeInTheDocument();
+      expect(queryByTestId('organization-nav')).toBeVisible();
```

Comment:
again, just out of curiosity, why are we changing the test from `toBeInTheDocument()` to `toBeVisible()`

Reply:
Also for consistency and I prefer `toBeVisible` since it's stricter than `toBeInTheDocument`